libcloud/test/backup/test_dimensiondata.py

```diff
@@ -35,19 +34,13 @@ def setUp(self):
         self.driver = DimensionData(*DIMENSIONDATA_PARAMS)
 
     def test_invalid_region(self):
-        try:
+        with self.assertRaises(ValueError):
             self.driver = DimensionData(*DIMENSIONDATA_PARAMS, region='blah')
-        except ValueError:
-            pass
 
     def test_invalid_creds(self):
         DimensionDataMockHttp.type = 'UNAUTHORIZED'
-        try:
+        with self.assertRaises(InvalidCredsError):
```

Comment:
Thanks for the ValueError tip, @Kami. That's actually the original reason why I used assertRaises as a context manager: so that I can inspect the error message while maintaining test code that looks exactly like how one would use the API in production:

```python
with self.assertRaises(ValueError) as err:
    DimensionData(*DIMENSIONDATA_PARAMS, region='blah')

self.assertEqual('Invalid region: blah', str(err.exception))
```

Unfortunately, I forgot to add the error message assertion. Heh. I'll add that in my next PR.